starter-code/index.html

```diff
@@ -3,28 +3,58 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" href="css/main.css">
 </head>
 <body>
-Premium Discover Help Download
+  <div class="body"> 
+  <nav>
+    <img src="images/spotify-logo.png" alt="">
+    <ul>
+      <li>Premium </li>
+      <li>Discover</li>
+      <li>Help</li>
+      <li>Download</li> 
+    </ul>
+  </nav>
 
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
+<div class="mainImg">
+<h1> Music for everyone.</h1>
+<p>Spotify is now free on mobile, tablet and computer. <br>
+  Listen to the right music, wherever you are.</p> 
+</div>
 
-What’s on Spotify?
+<div class="secPage">
 
-Millions of Songs
-There are millions of
+<header> 
```

Comment:
You should use a div instead, and call it the way you want, but usually the header is on the top of your webpage